### ISSUE TITLE
Express and Fastify cannot work together

### ISSUE THREAD

**Issue by gajus** [DataDog/dd-trace-js#1858]
**Describe the bug**

We are running a web service that (on different ports) is service HTTP requests using Express and Fastify.

It appears that `dd-trace` is only picking up express.js requests.

<img width="2056" alt="Screen Shot 2022-02-19 at 6 55 32 PM" src="https://user-images.githubusercontent.com/973543/154812940-6ff30e9a-8628-40b3-bd41-1d815aa345f4.png">

Not clear how to make it see the Fastify service in addition to Express.js

Furthermore, we had increase the max heap size as without it the app was crashing:

```yaml
- args:
    - '-c'
    - 'ts-node-transpile-only -r dd-trace/init -s ./dist/bin/server'
  command:
    - '/bin/sh'
  env:
    # Without increasing the limit, dd-agent is causing the following error:
    # https://gist.github.com/gajus/362e0f6b5b921a1766ba10c27b577d7d
    - name: NODE_OPTIONS
      value: '--max_old_space_size=4096'
```

**Environment**

* **Operation system:** Linux
* **Node version:** 16.13
* **Tracer version:** 2.2.0
* **Agent version:**

**Comment by rochdev** [comment]
Should this issue be merged with #1859?

**Comment by bengl** [comment]
Closing this in favour of the other issue.
